tests/queries/0_stateless/02320_projections_optimize_aggregation_in_order.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
```

Comment:
Could you rename the test to begin with `01710_`?

Reply:
OK (and I will add some clarification to existing `01710_projection_aggregation_in_order`)